packages/react-dom/src/__tests__/ReactUpdates-test.js

```diff
@@ -1734,4 +1739,57 @@ describe('ReactUpdates', () => {
       });
     }).toThrow('Maximum update depth exceeded');
   });
+
+  fit('prevents infinite update loop between a component with synchronous updates in useEffect and a component with synchronous updates in render', async () => {
```

Comment:
nit: should be `it`? Is there a lint rule that checks these?

Reply:
oh... this is a Draft PR. Sorry for these early comments, I'm still adjusting to this workflow.